core/src/main/java/com/squarespace/template/plugins/platform/ContentFormatters.java

```diff
@@ -267,6 +267,13 @@ private static String getFocalPoint(JsonNode media) {
   }
 
   private static String getAltTextFromContentItem(JsonNode contentItemNode) {
+
+    // Some blocks, such as image blocks, store alt text on the ContentItem
+    JsonNode altText = contentItemNode.path("altText");
+    if (isTruthy(altText)) {
```

Comment:
What if the customer leaves the `altText` value empty on purpose? Is this case handled correctly, i.e. is it surfaced as `alt=""` on the `<img>` tag, regardless of the values of `title`, etc.?

Reply:
It is not handled correctly, I'll fix this.

Reply:
What's the e.t.a on finishing this PR?

Reply:
Hi, sorry about the delay. I've been caught up in another project. I expect to have a fixed version by the end of the week. It also seems like a change in site server will be required, so I'll link that PR here when it's ready.